applications/plugins/SofaPython/Binding_BaseContext.cpp

```diff
@@ -92,37 +92,6 @@ static std::ostream& pythonToSofaDataString(PyObject* value, std::ostream& out)
         return out;
     }
 
-    if( PySequence_Check(value) )
```

Comment:
There was a long discussion in which we decided not to rely only on "repr/str" and instead allow objects to implement their own conversion operator to pass to the create function. Have a look for "getAsACreateObjectParameter" in Binding_BaseContext.cpp.

Removing the call as you do here suppress this behavior.